e2e/editor.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { test, expect, type Page } from '@playwright/test';
+
+let page: Page;
+
+test.beforeAll(async ({ browser }) => {
+  page = await browser.newPage();
+  await page.goto(
+    '/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-2'
+  );
+});
+
+test.afterAll(async () => {
+  await page.close();
+});
+
+test.describe('Editor Component', () => {
+  test('Should be clicked and able to insert text', async () => {
```

Comment:
```suggestion
  test('Should be clicked and able to insert text', async ({ page }) => {
```

We can get the `page` directly from the function args.